go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/kubernetes-csi/csi-test/v5
 go 1.18
 
 require (
-	github.com/container-storage-interface/spec v1.9.0
+	github.com/container-storage-interface/spec v1.9.1-0.20240305034746-abf6346bb13d
```

Comment:
We should use a released version.

Reply:
Yes, this one is the preview of the changes we need, for earlier review and CI.

I will update the PR when the release is ready.

Reply:
@huww98 The latest commit of the spec on `master` is now beyond the `abf6346bb13d` commit - it is at `2696773158b9` and contains the new CBT related `SnapshotMetadata` service.  No official CSI release available yet!